examples/sdkconfig.defaults

```diff
@@ -26,8 +26,7 @@ CONFIG_ESP32S3_SPIRAM_SUPPORT=n
 CONFIG_SPIRAM_SPEED_80M=y
 CONFIG_SPIRAM_IGNORE_NOTFOUND=y
 CONFIG_SPIRAM_USE_MALLOC=y
-CONFIG_HEAP_ABORT_WHEN_ALLOCATION_FAILS=y
-CONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=4096
+CONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=4090
```

Comment:
this is to avoid exhausting the internal ram when the SPIRAM is present

Reply:
In the grpc layer we allocate 4096 as a work buffer and I want this buffer in the SPIRAM when present otherwise we are at risk of an allocation failure in the internal ram and if this happens in rust code that will panic immediately. 
Normally this should be allocated only once when the webrtc server is instantiated (after the connection is established) but there is a bug in the grpc layer that causes this buffer to be moved and reallocated. 

The panic on allocation failure is not needed actually and it causes the program to fail if an allocation fails in esp-idf code.

we will have to update the template too